cmd/podman/parse.go

```diff
@@ -198,6 +198,11 @@ func readKVStrings(env map[string]string, files []string, override []string) err
 func parseEnv(env map[string]string, line string) error {
 	data := strings.SplitN(line, "=", 2)
 
+	// catch invalid variables such as `-e = ` or `-e =A`
+	if data[0] == "" {
+		return errors.Errorf("invalid environment variable: %q", line)
+	}
+
```

Comment:
maybe more tea time, but if I set line to "-e =A" (space between 'e' and '=') this is passing.  data[0] gets set to '-e ' (space after 'e') in that case.  I'm not sure this is doing what you're hoping for.  It also seems to pass for your other bad example `-e = `.

Reply:
The `-e` was not referring to the actual string but to the CLI option to add an env var, but I see why it can be interpreted that way. I'll fix that. Thanks!